### ISSUE TITLE
Load Layers: Williams 9 second substrate

### ISSUE THREAD

**Issue by Tasilee** [AtlasOfLivingAustralia/layers-service#107]
Email from Kristen Williams December 11, 2018. Updates to substrate layers at 9 second grid (~250m), see https://doi.org/10.25919/5bd6780e2fbc6.

These will be a replacement for all layers classified as "Substrate"

**Comment by RobinaSanderson** [comment]
There are 81 layers with classification "Substrate"

**Comment by Tasilee** [comment]
This will be an example where I would now decline the request as many of the existing and some proposed layers will be superseded by recommendations I have made in the review of existing layers. In this case, I am recommending that we use a subset of BoM's GeoFabric layers for substrate.

I have not, as yet, done a review of the outstanding 'Load Layer' issues as the discussion about strategy and decisions may reflect how the issues are viewed. In my monthly reports, I have sent criteria for evaluation of layer load requests and these are repeated here, but may (as implied above) be amended during the review process.

1. The information doesn’t already exist (e.g., IPAs are in CAPAD)
2. Is it an AREA (single or multi-polygon that could be imported/saved) rather than a layer?
3. National coverage, or international coverage where data helps place the Australian region in context
4. Presents information that suggests spatial control of species distributions or would be useful in the interpretation or management of species and areas
5. Data is from an authoritative source
6. Spatial resolution of 1km or better preferred
7. Fills a gap in or complements our layer collection (name, space, time, environmental factor)
8. Anticipated broad usage by one or more of the following communities (research, area managers, environmental consultants, citizen scientists, education, public)
9. Has quality metadata
10. Data is in a form that is readily ingested into our systems
11. Is up to date
12. Supersedes an existing layer (deprecation or deletion) 
13. Anticipates future requirements
14. Supports nationally significant projects (which themselves would need some evaluation, but GBR management given World Heritage status and threats would pass)
15. Openly available (free and open source data).
16. Externally funded? Probably relates to (14).

I would probably knock this issue out on 5, 7, 8, 9, 11, 13, 14

**Comment by RobinaSanderson** [comment]
Thanks @Tasilee  - So just to confirm: these layers should not be added, and the issue can be closed once Kristen is informed?

**Comment by Tasilee** [comment]
We won't decide on what layers in the backlog will be loaded until the review of layers completes. Once that is done, we will advise as needed.

**Comment by Tasilee** [comment]
BTW: There is a worksheet on Google Docs that lists all outstanding layer issues (similar to the SP worksheet). I updated it a few weeks ago, but I probably need to check it. This table makes it pretty easy to see what is there, priorities, effort but no 'criteria' as yet. I'd really like one of the developers to write some code to produce these tables. It should be pretty easy as I have a short Python program that extracts details of issues for the DQ tests work.

**Comment by peggynewman** [comment]
These layers weren't part of the rationalisation exercise. Closing.
